### PR TITLE
Major Update mission-and-core-values.md

### PR DESCRIPTION
**Pull request by ethereumnick**
Not so much an update as a reorganization with extensive additions.

### CODE REVIEW

**Comment by ethereumnick** [comment]
Cryptowanderer said:- 

> "In Mission, I think it is better to start each bullet with "To..", so would change point 3 to read "To foster links...". It just flows better imo.
> 
> I wouldn't capitalise Distributed and Decentralised in Beliefs (also Democracy later on). Also, pick decentralised, or decentralized, not both. I prefer the first.
> 
> Also, use Status, or Status.im (or make it crystal clear that one is the community and one the application)"

I agree with all three points

File: docs/getting-started/code-of-conduct.md [hunk 1]
```diff
@@ -1,95 +1,96 @@
 # Code of Conduct
 
-This page outlines how you should govern yourself when in our community. If you would like more practical tips and suggestions please read on [how to grow our Community](../community/how-to-grow-our-community.md).
+This page outlines how you should govern yourself when participating in the Status community. For more practical tips and suggestions please refer to [how to grow our Community](../community/how-to-grow-our-community.md).
```
Comment (0xc1c4da) [inline docs/getting-started/code-of-conduct.md:3]: ah the reason of using `our` here was deliberate, it's meant to personalise it and be inclusive.

File: docs/getting-started/code-of-conduct.md [hunk 2]
```diff
@@ -1,95 +1,96 @@
...
-    We expect participants in the project to resolve disagreements constructively. When they cannot, we escalate the matter to structures with designated leaders to arbitrate and provide clarity and direction.
-    
-- **Ask for help when unsure**
-Nobody is expected to be perfect in this community. Asking questions early avoids many problems later, so questions are encouraged, though they may be directed to the appropriate forum. Those who are asked should be responsive and helpful.
+- If you don’t know **Ask**
+Nobody is expected to be perfect in this community. Asking questions early avoids many problems later, so questions are encouraged, though they may be directed to the appropriate forum. Those who are asked should be responsive and helpful.  “RTFM” is not an appropriate response to an honest enquiry.
 
-- **Step down considerately**  
-When somebody leaves or disengages from the project, we ask that they do so in a way that minimises disruption to the project. They should tell people they are leaving and take the proper steps to ensure that others can pick up where they left off.
+- Be **Respectful**
```
Comment (0xc1c4da) [inline docs/getting-started/code-of-conduct.md:31]: Be outside ** ?

**Comment by 0xc1c4da** [comment]
thanks @ethereumnick for the contribution! apologies to being so late with this

**Pull request merged**